### ISSUE TITLE
New maintainer for munch

### ISSUE THREAD

**Issue by rik-x777** [Matrixx-Devices/official_devices#92]
## Name and Codename of the device you want to apply for
```
poco f4 (munch)
```

## Device Tree sources

```
https://github.com/rik-x777/device/tree/matrixx
https://github.com/rik-x777/common
https://github.com/AnkitSavani111/android_kernel_xiaomi_sm8250.git
https://github.com/TheMuppets/proprietary_vendor_xiaomi_munch.git
https://github.com/TheMuppets/proprietary_vendor_xiaomi_sm8250-common.git
```

## What ROM's do you maintain officially/unofficially
```
crdroid officially and rising os unofficially.
```

## How long have you been building Custom ROMs
```
2 months
```

## Do you build locally or rely on server if server what cloud service provider do you use ?
```
server. crave.io
```
## Free trial user or paid user?
```
free
```

### Telegram username
@rik_x290

### Telegram username
Rik

### GitHub username
rik-x777

### [MUST] Link to your XDA thread of unofficial build
https://xdaforums.com/t/rom-android-14-qpr3-project-matrixx-10-9-0-community-munch.4698406/#post-89763944

### [MUST] Link to your unofficial Matrixx Build Post other than XDA
https://t.me/PocoF4updates/1203

**Comment by kshitij-vk** [comment]
The application was already rejected by the team. Re-opening the issue will not help unless you plan to work on your trees and learn about how things work. For now a person looking at your trees would only see bring-up commits from Evox to Matrixx. This does not showcase your skills. Take a look at device trees present in matrixx org. The maintainers have done their own research and did changes to show what they now. Which is not the case here. Please do not reopen issues unless that missing thing is added.

**Comment by rik-x777** [comment]
@Roxor-007 the trees already includes all of the necessary things that are needed. the only thing that I found to be missing was kprofile front end and the encryption which I did changed. and the previous issue had different trees. and all of the trees at least for munch, exists as a fork of someone without any major changes. if by missing things, you mean to say that I will need to add all of the stuffs in device and common trees manually without forking someone else's trees, it will just be a taxing manual labor without providing any actual benefit to the rom's usage
